SCClassLibrary/Common/Files/SoundFile.sc

```diff
@@ -422,7 +422,9 @@ SoundFile {
 	asBuffer { |server|
 		var buffer, rawData;
 		server = server ? Server.default;
-		if(server.serverRunning.not) { Error("SoundFile:asBuffer - Server not running.").throw };
+		if(server.serverRunning.not) { 
+			Error("SoundFile:asBuffer - Server not running.").throw
+			"server '%' not running".format(server.name).warn; };
```

Comment:
I don't think we should change this one. The warning wouldn't probably even be posted after throwing an error. I'd just leave it as it was.

Reply:
Donw in https://github.com/supercollider/supercollider/pull/6796/commits/6402765df63066721da49fba102a46a0d6571baa!